qa/rpc-tests/ctor.py

```diff
@@ -233,7 +234,7 @@ def Test():
 
     # Execution is much faster if a ramdisk is used, so use it if one exists in a typical location
     if os.path.isdir("/ramdisk/test"):
-        flags.append("--tmpdir=/ramdisk/test/ma")
+        flags.append("--tmpdir=/ramdisk/test/t")
```

Comment:
?

Reply:
t for test :-)  You are correct, its an irrelevant change...